bin/run_brick.rb

```diff
@@ -13,17 +14,22 @@
 
 log.info "action=#{brick_type}_execution status=init"
 
+def get_brick_params(prefix)
+  ENV.select {|k,| k.to_s.match(/^#{prefix}.*/)}.map {|k, v| [k.slice(prefix.length..-1), v]}.to_h
```

Comment:
warning: Layout/SpaceInsideBlockBraces: Space between { and | missing. (rubocop)